backend/open_webui/routers/ollama.py

```
@@ -1694,25 +1648,27 @@ async def download_file_stream(
+
+                            if response.ok:
+                                res = {
+                                    "done": done,
+                                    "blob": f"sha256:{hashed}",
+                                    "name": file_name,
+                                }
+                                os.remove(file_path)
+
+                                yield f"data: {json.dumps(res)}\n\n"
+                            else:
+                                raise "Ollama: Could not create blob, Please try again."
```

Ruff (0.14.7)

Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)